crates/sui-bridge/src/sui_client.rs

```diff
@@ -258,8 +305,28 @@ impl SuiClientInner for SuiSdkClient {
             .await
     }
 
-    async fn get_bridge_committee(&self) -> Result<BridgeCommittee, Self::Error> {
-        unimplemented!()
+    async fn get_bridge_committee(&self) -> Result<MoveTypeBridgeCommittee, Self::Error> {
+        let object_id = *get_bridge_object_id();
+        let resp = self
+            .read_api()
```

Comment:
You may need to introduce a new bridge-specific API that allows you to return the inner object instead of the bridge object.